src/main/frontend/extensions/code.cljs

```diff
@@ -100,6 +101,9 @@
         "csharp" "text/x-csharp"
         "objective-c" "text/x-objectivec"
         "scala" "text/x-scala"
+        "js" "text/javascript"
+        "ts" "text/typescript"
```

Comment:
Should we add `"typescript" "text/typescript"` here too?

Reply:
done